### PR TITLE
Upgrade yargs to 15 and jest to 25

### PR DESCRIPTION
**Pull request by vsumner**
This PR upgrades:

- `yargs` and `@types/yargs` to version `15` from `11`
- `jest` from `24` to `25`

See [Yargs Changelog](https://github.com/yargs/yargs/blob/master/CHANGELOG.md)
[Jest Changelog](https://github.com/facebook/jest/blob/master/CHANGELOG.md)

### CODE REVIEW

File: packages/graphql-typescript-definitions/package.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
     "prepublishOnly": "yarn build"
   },
   "devDependencies": {
-    "@types/chalk": "^2.2.0",
```
Comment (alexandcote) [inline packages/graphql-typescript-definitions/package.json:28]: We don't need `@types/chalk` finally? 😃

File: packages/graphql-typescript-definitions/src/cli.ts [hunk 2]
```diff
@@ -55,12 +54,16 @@ const argv = yargs
...
 
+type Args = typeof argv;
+
 const builder = new Builder({
   cwd: argv.cwd,
-  schemaTypesPath: argv.schemaTypesPath,
-  addTypename: argv.addTypename,
-  enumFormat: argv.enumFormat,
-  customScalars: normalizeCustomScalars(argv.customScalars),
+  schemaTypesPath: argv.schemaTypesPath as Args['schema-types-path'],
```
Comment (alexandcote) [inline packages/graphql-typescript-definitions/src/cli.ts:61]: I think I prefer doing something like this instead.
```
const builder = new Builder({
  cwd: argv.cwd,
  schemaTypesPath: argv['schema-types-path'],
  addTypename: argv['add-typename'],
  enumFormat: argv['enum-format'],
  customScalars: normalizeCustomScalars(
    argv['custom-scalars'],
  ),
});
```

Any strong opinion?

**Review by alexandcote — APPROVED** [review]

File: packages/graphql-typescript-definitions/package.json (hunk 1 above)
Comment (vsumner) [reply packages/graphql-typescript-definitions/package.json:28]: It's a stub now. Chalk includes its own types.

File: packages/graphql-typescript-definitions/src/cli.ts (hunk 2 above)
Comment (vsumner) [reply packages/graphql-typescript-definitions/src/cli.ts:61]: Yeah that's better.

**Pull request merged**